src/standalone/Snow/Snow.jl

```diff
@@ -236,11 +249,27 @@ auxiliary_domain_names(::SnowModel) = (
     :surface,
     :surface,
     :surface,
+    :surface,
+    :surface,
+    :surface,
 )
 
 
 ClimaLand.name(::SnowModel) = :snow
 
+"""
+    scf(x::FT; α = FT(1e-3))::FT where {FT}
+
+Returns the snow cover fraction, assuming it is a heaviside
+function at 1e-3 meters.
+
+In the future we can play around with other forms, this makes it a bit more
+explicit.
+"""
+function scf(x::FT; α = FT(1e-3))::FT where {FT}
+    return heaviside(x - α)
+end
+
 function ClimaLand.make_update_aux(model::SnowModel{FT}) where {FT}
     function update_aux!(p, Y, t)
```

Comment:
This function does not care if about other components and is only dependent on the snow state.